src/lib.rs

```diff
@@ -59,49 +94,64 @@ impl Command {
             .into_iter()
             .map(|s| s.as_ref().to_string())
             .collect::<Vec<String>>();
-        Ok(Command {
+        Ok(T::insert_command(Command {
             name: name.as_ref().to_string(),
             command: command.as_ref().to_string(),
```

Comment:
This can save you an allocation when the caller provides a `String` instance for `command`.


```suggestion
            command: command.into(),
```